fs/remote/resolver.go

```diff
@@ -91,11 +92,11 @@ func NewResolver(cfg config.BlobConfig, handlers map[string]Handler) *Resolver {
 	if cfg.MaxRetries == 0 {
 		cfg.MaxRetries = defaultMaxRetries
 	}
-	if cfg.MinWaitMSec == 0 {
-		cfg.MinWaitMSec = defaultMinWaitMSec
+	if cfg.MinWaitMsec == 0 {
+		cfg.MinWaitMsec = defaultMinWaitMSec
```

Comment:
I'm okay with either format but we should keep them consistent (e.g. `MinWaitMsec` -> `MinWaitMSec`, or `defaultMinWaitMSec` -> `defaultMinWaitMsec`, same for `defaultMaxWaitMSec`).